engine/tests/new-tests/apply_filter.cu

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/cudf.h>
+#include <cudf/types.hpp>
+#include <from_cudf/cpp_tests/utilities/base_fixture.hpp>
+#include <cudf/table/table_view.hpp>
+#include <cudf/sorting.hpp>
+#include <cudf/copying.hpp>
+#include <cudf/column/column_factories.hpp>
+#include <from_cudf/cpp_tests/utilities/column_utilities.hpp>
+#include <cudf/utilities/type_dispatcher.hpp>
+#include <from_cudf/cpp_tests/utilities/type_lists.hpp>
+#include <from_cudf/cpp_tests/utilities/column_wrapper.hpp>
+#include <from_cudf/cpp_tests/utilities/legacy/cudf_test_utils.cuh>
+#include <from_cudf/cpp_tests/utilities/table_utilities.hpp>
```

Comment:
How we are going to sync code for #include <from_cudf/...> when the original cudf repo is updated?